config/locales/fr.yml

```diff
@@ -261,18 +312,12 @@ fr:
     distance_in_words:
       about_x_hours: "%{count}h"
       about_x_months: "%{count}mois"
-      about_x_years:
-        one: un an
-        other: "%{count} ans"
-      almost_x_years:
-        one: un an
-        other: "%{count} ans"
-      half_a_minute: À l’instant
+      about_x_years: "%{count}an(s)"
+      almost_x_years: "%{count}an(s)"
+      half_a_minute: À l'instant
```

Comment:
On est sûr que cela ne pose pas de soucis d'échappement que d'avoir ça sans guillemets? (même remarque pour la ligne 319)

Reply:
Il n'y a pas de guillemets dans la version anglaise : https://discourse.joinmastodon.org/t/use-a-software-to-help-us-translate-mastodon/470